lms/djangoapps/instructor/utils.py

```diff
@@ -186,7 +187,13 @@ def collect_ora2_data(course_id, include_email=False):
     cursor.execute(raw_queries[0])
     cursor.execute(raw_queries[1], [course_id])
     header = [item[0] for item in cursor.description]
-    return header, cursor.fetchall()
+    header.append('formatted_answer')
+    raw_answer_index = header.index('raw_answer')
+    data = cursor.fetchall()
+    data_rows = [list(row) + [dumps(
```

Comment:
And just as a general note, Python comprehensions are easiest to follow with the format:

```python
comprehension = [
    item
    for item in collection
    if condition
]
# where condition is an optional boolean expression
```